src/css/profile/mobile/common/progress.less

```diff
@@ -30,21 +47,22 @@ tau-progress {
 }
 
 .ui-progress-bar-value-bg {
-	min-height: 3 * @px_base;
+	height: 3 * @px_base;
+	max-height: 3 * @px_base;
 	width: 100%;
 	position: relative;
 	flex: 1;
 	order: 2;
 	background-color: var(--progress-bar-bg-color);
-	border-radius: 3 * @px_base;
+	border-radius: 1.5 * @px_base;
 }
 .ui-progress-bar-label {
 	span ~ span {
-		margin-left: 4 * @unit_base; // it doesn't defined in guideline
+		margin-left: 4 * @px_base; // it doesn't defined in guideline
```

Comment:
When you are changing these units you have to make division by 2.

Reply:
OK, Done